src/cloudwatch-mcp-server/awslabs/cloudwatch_mcp_server/cloudwatch_metrics/tools.py

```diff
@@ -792,30 +881,242 @@ def _parse_alarm_recommendation(self, alarm_data: Dict[str, Any]) -> AlarmRecomm
+        - Seasonality detection
+        - Trend analysis
+        - Data density and publishing period
+        - Advanced statistical measures (min/max/median, std dev, noise)
+
+        Usage: Use this tool to get objective metric analysis data. For alarm recommendations,
+        use the get_recommended_metric_alarms tool instead.
+
+        Args:
+            ctx: The MCP context object for error handling and logging.
+            namespace: The metric namespace (e.g., "AWS/EC2", "AWS/Lambda")
+            metric_name: The name of the metric (e.g., "CPUUtilization", "Duration")
+            dimensions: List of dimensions with name and value pairs
+            region: AWS region to query. Defaults to 'us-east-1'.
+            analysis_period_minutes: Number of minutes of data to analyze. Defaults to 2 weeks (20160 minutes).
+
+        Returns:
+            Dict[str, Any]: Raw analysis data including seasonality, trend, density, and statistics only.
+
+        Example:
+            analysis = await analyze_metric(
+                ctx,
+                namespace="AWS/EC2",
+                metric_name="CPUUtilization",
+                dimensions=[
+                    Dimension(name="InstanceId", value="i-1234567890abcdef0")
+                ]
+            )
+            print(f"Seasonal strength: {analysis['seasonality_seconds']['seasonal_strength']}")
+            print(f"Trend direction: {analysis['trend']['trend_direction']}")
+            print(f"Data quality: {analysis['data_quality']['quality_score']}")
+        """
+        try:
+            if analysis_period_minutes <= 0:
+                raise ValueError('analysis_period_minutes must be positive')
+
+            logger.info(
+                f'Analyzing metric: {namespace}/{metric_name} using period of {analysis_period_minutes} minutes'
+            )
+
+            end_time = datetime.now(timezone.utc)
+            start_time = end_time - timedelta(minutes=analysis_period_minutes)
+
+            metric_data_response = await self.get_metric_data(
+                ctx=ctx,
+                namespace=namespace,
+                metric_name=metric_name,
+                dimensions=dimensions,
+                start_time=start_time.isoformat(),
+                end_time=end_time.isoformat(),
+                statistic='Average',
+                region=region,
+                target_datapoints=analysis_period_minutes,
+            )
+
+            # Parse response into structured data
+            _, _, period_seconds = self._prepare_time_parameters(
+                start_time, end_time, analysis_period_minutes
+            )
+            metric_data = self._parse_metric_data_response(metric_data_response, period_seconds)
+            analysis_result = self.metric_analyzer.analyze_metric_data(metric_data)
+
+            analysis_result.update(
+                {
+                    'recommendations_allowed': False,
```

Comment:
Do we really need this? Should we rename it into `alarm_recommendations_allowed`?